components/card/card.css

```diff
@@ -0,0 +1,109 @@
+:root{
+    --card-primary-color: #2a9d8f;
+    --card-font-color : #fff;
+    --card-border-color : #bacdd8;
+    --card-dark-color : #000;
+    --card-status-color : #807F7F;
+}
+.card-container {
+    max-width: 230px;
+    min-width: 220px;
```

Comment:
You should not use pixel here.

Reply:
yes don't use pixels it will create problems in responsiveness.

Reply:
changed to em